src/crud/document.py

```
@@ -194,27 +249,150 @@ async def create_documents(
                 )
-            )
+
+            if doc.embedding:
+                new_doc.sync_state = "pending"
+            honcho_documents.append(new_doc)
+
+            # Track embedding for vector store (ID will be available after commit)
+            if doc.embedding:
+                docs_with_embeddings.append((new_doc, doc.embedding))
+
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)